rllib/algorithms/sac/tests/test_sac.py

```diff
@@ -151,12 +151,14 @@ def test_sac_compilation(self):
                     new_trainer = sac.SACTrainer(config, env=env)
                     new_trainer.restore(checkpoint)
                     # Get some data from the buffer and compare.
-                    data = trainer.local_replay_buffer.replay_buffers[
+                    storage = trainer.local_replay_buffer.replay_buffers[
                         "default_policy"
-                    ]._storage[: 42 + 42]
```

Comment:
Slicing should still be supported, right?
I know that it does not seem super relevant here but it's an operation that will definitely come in very handy for upcoming changes. @fwitter